submissions/4zorbin/html-css-popup/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Hooli-pop</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <nav>
+      <ul class="nav">
+        <li><a href="#" class="nav--button">Mail</a></li>
+        <li><a href="#" class="nav--button">Images</a></li>
+        <li>
+          <input type="checkbox" id="toggle" />
+          <label for="toggle" class="check-button"
+            ><img src="icons/popup-button.png" alt="Menu"
+          /></label>
```

Comment:
I think, that `>` closing quote must be on one line with your `<` start quote.

Reply:
done